apps/cnquery/cmd/plugin.go

```diff
@@ -120,7 +120,7 @@ func (c *cnqueryPlugin) RunQuery(conf *run.RunQueryConfig, runtime *providers.Ru
 
 	for i := range assets {
 		connectAsset := assets[i]
-		connectAssetRuntime, err := providers.Coordinator.RuntimeFor(connectAsset, runtime)
+		connectAssetRuntime, err := providers.Coordinator.EphemeralRuntimeFor(connectAsset)
```

Comment:
ooh, this must be the provider switch fix?

Reply:
No, that's for the cleanup.

I think, I need to test this with multiple assets and a `run` command. Not sure how this would behave.

I didn't need a provider switch fix for ebs. But I don't understand why.

Reply:
ah, i just tested this, i think we're still getting the wrong provider, we shouldn't be hitting describe instances before we hit some other logs, looks like we're redirecting this to the aws provider

Reply:
i tried testing with your other pr and it somehow didn't fix the connection here...ill dig in today